docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.md

```diff
@@ -43,7 +46,12 @@ options.setProfile(profile);
 driver = new RemoteWebDriver(options);
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-# We don't have a Python code sample yet -  Help us out and raise a PR
+from selenium.webdriver.firefox.options import Options
+from selenium.webdriver.firefox.firefox_profile import FirefoxProfile
+options=Options()
+firefox_profile = FirefoxProfile(profile)
```

Comment:
No need to create a `FirefoxProfile` to give to options, it does it for you if you pass a string.